tests/test_intermediate.py

```
@@ -168,3 +170,47 @@ def test_400_decrypt_intermediate(
+        )
+
+        assert r.status_code == status.HTTP_404_NOT_FOUND
+        assert detail_of(r) == f"Temp bucket for analysis with ID {analysis_id} was not found"
+    finally:
+        reset_client()
+
+
+def test_502_no_single_file(test_client, rng, analysis_id, remote_node_and_private_key):
+    reset_client = temporarily_change_dependency(test_client, get_storage_client, lambda: MockClient())
+    blob = next_random_bytes(rng)
+    remote_node, remote_private_key = remote_node_and_private_key
```

Ruff (0.15.10)

[warning] Unpacked variable `remote_private_key` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)